frontend/src/scenes/notebooks/Nodes/utils.tsx

```diff
@@ -15,6 +15,10 @@ export function reportNotebookNodeCreation(nodeType: string): void {
     posthog.capture('notebook node created', { type: nodeType })
 }
 
+export function removeQueryParams(input: string): string {
```

Comment:
i've no idea what input is coming in here... some of them were being forced to particular types like insight short id or number

should this be taking `input: unknown` 
should we be converting to a URL before removing search params instead of splitting on question mark

only questions sorry 🙈

Reply:
It's hard to test the paste handling exactly because it's tightly coupled to TipTap but hopefully the regex tests at least show the intention here